### ISSUE TITLE
tmux server exited unexpectedly

### ISSUE THREAD

**Issue by link-ding** [tmux/tmux#2376]
### Issue description
I used `tmux kil-serverl` to exit tmux and then terminal shows server exited unexpectedly.
### Required information

Please provide the following information:

* tmux version (`tmux -V`) : tmux 3.1b
* Platform (`uname -sp`) : Darwin i386
* $TERM inside and outside of tmux (`echo $TERM`). outside: xterm-256color inside: I can not use tmux now
* Logs from tmux (`tmux kill-server; tmux -vv new`). This also gives 'server exited unexpectedly'

**Comment by nicm** [comment]
Works OK for me. Does this still happen with master?


If so then please show logs from doing `tmux kill-server; tmux -vv new` then `tmux kill-server`.

**Comment by chhschou** [comment]
@link-ding I also cannot start tmux with the `server existed unexpectedly` msg. I fixed it by remove files starting with `tmux` in `/tmp`

**Comment by casprwang** [comment]
@chhschou  
Works like a charm. Curious how did you find/debug this solution. Some cache stored there was interrupting the tmux server start?

**Comment by chhschou** [comment]
> @chhschou  
> 
> Works like a charm. Curious how did you find/debug this solution. Some cache stored there was interrupting the tmux server start?

Hi @casprwang I got this issue after upgrade to tmux 3.1b from 2.9 so I uninstalled and went back to 2.9. The issue still persisted with 2.9 which didn't before so I start to look into /tmp and other file caches, deleting files that looks related to tmux then try  restarting tmux until it started. The author of tmux could perhaps document (in a faq or something easy to read) what files are used by tmux so user can do a proper clean uninstall.
